renovate.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "extends": ["config:js-app", "group:allNonMajor"],
+  "extends": ["config:js-lib", "group:allNonMajor"],
```

Comment:
So we should do this for the other repos as well?

Reply:
Only for repos that are used as a dependency in other package.json files. Like this one, report-monitoring and reporting-dimensions (so that they don't force the consuming project to use specific versions of transitive dependencies).

For services and reports, "js-app" (ie. pinning all dependencies) is still a good choice.